libs/@local/hash-graph-sdk/python/scripts/generate_filter.py

```diff
@@ -0,0 +1,446 @@
+"""Generate `graph_sdk.filter.path` from `graph_client.models`.
+
+We need to generate 4 classes:
+
+- DataTypePath
+- PropertyTypePath
+- EntityPath
+- EntityTypePath
+
+To generate them we have a configuration file that is located
+under `graph_sdk/filter/config.json`.
+
+The configuration file is a record with the following fields:
+- `data_type`
+- `property_type`
+- `entity_type`
+- `entity`
+
+Each entry has the following fields:
+- `selector`: Fields that require an intermediary `*` selector, and then continue
+              with another type. It is a record, where the key is the field name
+              and the value is the type to continue with.
+              (one of `data_type`, `property_type`, `entity_type`, `entity`)
+- `wildcard`: Fields that go into untyped territory through the properties selector.
+- `continue`: Fields that do not require an intermediary `*` selector, and then continue
```

Comment:
Maybe better named as `direct`? `continue` sounds like a control flow.

Reply:
renamed in [c6abea7](https://github.com/hashintel/hash/pull/2878/commits/c6abea70ba0c598f42a7e1651d2bc6161202a104)